test/utils.tsx

```diff
@@ -37,4 +40,16 @@ export const buildRenderForm = (initialValues: Record<string, unknown>) => {
         ),
     };
   };
+  return renderForm;
+};
+
+export const changeValue = async (
+  el: HTMLInputElement,
+  eventName = 'vaChange'
+) => {
+  el.value = 'asdf';
```

Comment:
This is a bit picky, but can the value be passed in as an argument instead of hard-coded? I can imagine wanting to change the value to something specific for _test reasons_. Particularly if we were to reuse this utility function on a radio button.

Reply:
Not picky at all! 100% should be customizable. I just never got back around to it. Good call. :+1: